request/js/main.js

```diff
@@ -0,0 +1,109 @@
+$(function() {
+
+    // QUESTIONS (by default the form starts by asking whether the request is a referral or not)
+    // Will most likely need to be moved to a JSON file at some point in the near future
+    var referralQuestionList = [
+      {"key":"name", "value": "Their Name", "helper": "What's their name?"},
+      {"key":"referer_name", "value": "Your Name", "helper": "What's your name?"},
+      {"key":"age", "value": "Their Age", "helper": "How old are they?"},
+      {"key": "gender", "value": "Gender", "helper": "What gender do they identify as?"},
+      {"key": "school", "value": "School Name", "helper": "What school do they attend?"},
+      {"key": "contact", "value": "Their Contact", "helper": "What's the best way to reach them?"},
+      {"key": "referer_contact", "value": "Your Contact", "helper": "What's the best way to reach you?"},
+      {"key": "situation", "value": "Please Explain", "helper": "Please provide any additional information."}
+    ]
+    var individualQuestionList = [
```

Comment:
I would agree. Some of this stuff is a little hacky as some of it was prototypical in nature.